commands-yml/commands/context/get-contexts.yml

```diff
@@ -8,6 +8,9 @@ description:
     For information on the format of the context names, see the [get context
     documentation](/docs/en/commands/context/get-context.md).
 
+    On iOS, using the XCUITest driver, one can use the `mobile: getContexts`
```

Comment:
maybe:

`On iOS, using the XCUITest driver, one can use the mobile: getContexts command as an alternative to the standard method in order to get the title and url associated with each context as additional metadata`.

Just want to be clear users should think of it as a non-standard alternative to the regular one.